tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
 	"compilerOptions": {
 		"checkJs": true,
-		"noEmit": true
+		"noEmit": true,
+		"module": "ESNext",
```

Comment:
are these tsconfig changes safe or could they cause dts buddy output changes or other shenanigans that require changesets?

Reply:
This the root TS config which isn't used for any of the packages, it's purely internal; if there were DTS buddy changes due to this CI would fail